client/src/config.ts

```diff
@@ -11,3 +11,7 @@ export const STATIC_URL =
 export const S3_URL = 'https://csgo-stratbook.s3.amazonaws.com/';
 
 export const TOKEN_TTL = 1 * 60 * 60 * 1000; // 1h
+
+export const SPLITBEE_ID = 'J3KX6SRRBPBD';
+
+export const GA_ID = 'G-787NDTZVPN';
```

Comment:
Shall we inject this via .env, in the future?

Reply:
Yes, most definately. Afaik, on the nest_backend branch the FE already uses .env instead of this TS file.